src/index.js

```diff
@@ -153,7 +153,9 @@ export function configure(env) {
 			],
 
 			alias: {
-				preact: path.resolve(__dirname, 'preact.js'),
+				preact: path.resolve(__dirname, 'shims/preact.js'),
+				'preact-router': path.resolve(__dirname, 'shims/preact-router.js'),
+				'react-apollo': path.resolve(__dirname, 'shims/react-apollo.js'),
```

Comment:
Other obvious one that jumps out at me is `preact-redux` that we might want to add.